README.md

```diff
@@ -260,31 +260,57 @@ makes it very easy to compare edits.
    ```
    git rebase upstream/master -i
    ```
-1. If you have messy commits, flag them with the squash thing. If there's only one
-conflicing commit, just `ctrl-x` out of the interactive window/message. You 
-should see an error message saying that the patch couldn't be applied.
+
+1. You will be brought into your system editor (depends on your configuration, could be `vim`, 
+   `nano` or other ones), with instruction from `git`.
```

Comment:
I find these instructions pretty hard to follow.  Step 1, ok, so now I am in the editor, but after that, what am I seeing?  What am I typing?  What is the workflow?  I don't really have any idea with just 
```
pick ....
```